tests/integration_tests/functional/test_vsock.py

```diff
@@ -38,7 +38,11 @@ def test_vsock(
         bin_vsock_path,
         test_fc_session_root_path
 ):
-    """Vsock tests. See the module docstring for a high-level description."""
+    """
+    Vsock tests. See the module docstring for a high-level description.
```

Comment:
Check guest and host vsock initiated connectins. Check the module docstring for details on the setup.

Reply:
done